gulpfile.js

```diff
@@ -8,16 +8,24 @@ const eslint = require('gulp-eslint');
 const mocha = require('gulp-mocha');
 const sourcemaps = require('gulp-sourcemaps');
 
-const babelClean = () => del(['build']);
+/**
+ * @summary Delete the dist/ directory
+ */
+const babelClean = () => del(['dist']);
 
+/**
+ * @summary Copy files that don't need to be compiled from src/ to dist/
+ */
 const babelCopy = () => gulp.src(['src/**', '!src/**/*.js', '!src/tests/integration/**'])
-  .pipe(gulp.dest('build/dist'));
+  .pipe(gulp.dest('dist'));
```

Comment:
`dist` has been used many times among lots of files, do you think we should store it into somewhere like config file?

Reply:
This shouldn't be an issue now since the files are stored in `build/`

Reply:
From our conversation, we probably don't want to add this to a config file.
* If we specify a build directory that is 2 or more directories deep, this could break some of our file paths
* If our unit tests reference a file using the build directory, they couldn't be run without a config file, but ideally, we shouldn't need one